### DISCUSSION TITLE
netbox 3.0 / Searching for the Customization area in the admin panel

### DISCUSSION THREAD

**Discussion by jchnbckr** [netbox-community/netbox#7136] [Q&A] [answered]
Hello,
i upgraded vom 2.11 to 3.0, in the morning to 3.0.1 and I am missing the Customization area in the admin panel. I want to create some custom attributes but the links are away.
Have I missed something by migration?
Greatings

**Comment by ziggekatten** [comment, marked as answer]
Custom links, export templates and fields are now managed directly in the Front end under "Other" and not in the admin section.

**Comment by jchnbckr** [comment]
Thx that was to trivial
